login.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/login.css">
+    <title>login</title>
+</head>
+<body>
+    <div class="login_box">
+        <div class="logo_img"><a href="/index.html"><img src="img/lg.png" alt="판다마켓"></a></div>
+        <div class="login_text">
+            <p class="login_tx">이메일</p>
+            <input class="login_input" type="text" placeholder="이메일을 입력해주세요">
```

Comment:
❗️ 수정요청
특정 input이 어떤 값을 받는지 설명하는 text들을 label 이라고 합니다.
이러한 친구들은 일반적으로 마우스로 선택시 연결된 input으로 focus가 이동하는 기능이 있습니다~
이를 위해, 그리고 명확한 연결관계 표시하기 위해 label 태그를 사용해주시는 것이 좋습니다~

```suggestion
            <label class="login_tx" for="email">이메일</label>
            <input id="email" class="login_input" type="text" placeholder="이메일을 입력해주세요">
```

label과 input이 연결하기 위해 label의 for 속성에 input의 id를 넣어주어야 합니다.